packages/xod-cli/src/commands/tabtest.js

```diff
@@ -0,0 +1,132 @@
+import path from 'path';
+import { exit } from 'process';
+import { tmpdir } from 'os';
+import { append, curry, identity, map, pick, tap, toPairs } from 'ramda';
+import fs from 'fs-extra';
+import { flags } from '@oclif/command';
+
+import childProcess from 'child_process';
+import { foldEither, allPromises } from 'xod-func-tools';
+import { loadProject } from 'xod-fs';
+import * as Tabtest from 'xod-tabtest';
+
+import BaseCommand from '../baseCommand';
+import commonArgs from '../args';
+import * as myFlags from '../flags';
+import {
+  BUNDLED_WORKSPACE_PATH,
+  BUNDLED_TABTEST_WORKSPACE_PATH,
+  BUNDLED_TABTEST_SRC_PATH,
+  BUNDLED_CATCH2_PATH,
+} from '../constants';
+import * as msg from '../messages';
+
+const spawn = (cmd, args, opts) =>
+  new Promise((resolve, reject) => {
+    childProcess.spawn(cmd, args, opts).on('exit', code => {
```

Comment:
Recently, I had a bug with `spawn` on Windows (especially, if the path contains spaces).
Did you test it?

Reply:
Just to subproccesses here: `make` and `make test`.